clang/lib/Format/ContinuationIndenter.cpp

```diff
@@ -1457,6 +1457,11 @@ unsigned ContinuationIndenter::getNewLineColumn(const LineState &State) {
       !Current.isOneOf(tok::colon, tok::comment)) {
     return ContinuationIndent;
   }
+  if (Style.isCpp() && Current.is(tok::arrow) &&
```

Comment:
```suggestion
  if (Style.isCpp() && Current.is(TT_TrailingReturnArrow) &&
```
This is what you need, right?